### DISCUSSION TITLE
Add a tooltip to the new buttons (Swap, Send, Receive)

### DISCUSSION THREAD

**Discussion by Kolibri1990** [DeFiCh/app#848] [Ideas]
Hello together,

In my oppinion for new users it would be helpfull to have tooltips on the buttons.
They don't know what this could be before clicking them
 -> https://user-images.githubusercontent.com/66674482/117842854-9c218d00-b27e-11eb-9413-3dd4bb2aff0c.png

Maybe Swap , Send and Receive is enough.

**Comment by nippur72** [comment]
I had the same thought 😄 see #831

**Comment by John-Gee** [comment]
I sort of disagree.

I think if most people can't easily tell what those buttons are for, then the buttons don't need tooltips, they need better icons.

**Comment by thedoublejay** [comment]
Hi @Kolibri1990 , thanks for this suggestion. We have added some tooltip guide on most buttons for better usability. You can see them on the latest app version 2.4.2. - https://defichain.com/downloads/
